Resources/Prototypes/GameRules/events.yml

```
@@ -186,7 +186,7 @@
     - id: KingRatMigration
     - id: NinjaSpawn
     - id: ParadoxCloneSpawn
-    - id: RevenantSpawn
+    #- id: RevenantSpawn
```

Needs to have a comment similar to this structure:
```suggestion
    #- id: RevenantSpawn # Omu - Removed from midround events.
```